polaris-react/src/components/LegacyCard/LegacyCard.scss

```diff
@@ -66,7 +66,7 @@
     padding: var(--p-space-5);
 
     #{$se23} & {
-      padding: var(--p-space-4);
+      padding: var(--p-space-2) var(--p-space-4);
```

Comment:
This gives sections a top and bottom padding of `8px` so when two sections are together, it will create a `16px` spacing. We have to do this instead of putting all `16px` on one side of so subdued sections and normal sections have even spacing between them

example:

<img width="483" alt="Screenshot 2023-06-22 at 1 04 00 PM" src="https://github.com/Shopify/polaris/assets/20652326/2752d3fb-936b-4538-b819-fe1b545df059">